app/frontend/styles/application.scss

```diff
@@ -16,3 +16,17 @@ $govuk-image-url-function: frontend-image-url;
 .autocomplete__wrapper, .autocomplete__input, .autocomplete__hint {
   font-family: $govuk-font-family;
 }
+
+.app-styled-content {
```

Comment:
I don't understand this class name – what does it mean?

Reply:
It's to wrap HTML which contains plain tags such as `h1, h2, p` and automatically style them with GOV.UK typography, commonly used for convenience, in this case we're using it for the markdown section

Reply:
`app` is the namespace for everything in this app (as opposed to `govuk`, which is CSS from govuk-frontend). I've moved this into its own file, maybe it's clearer?

Reply:
Ah, `app-` is equivalent to `govuk-`. That helps, thank you :+1: